src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -6182,6 +6195,64 @@ theorem toNat_lt_two_pow_sub_clz {x : BitVec w} :
         · simp [show w + 1 ≤ i by omega]
       · simp; omega
 
+/-! ### Count trailing zeros -/
+
+theorem ctz_eq_reverse_clz {x : BitVec w} :
+    x.ctz = (x.reverse).clz := by
+  simp [ctz]
```

Comment:
Do we also want a theorem in the other direction? `clz` -> `reverse ctz`?

Reply:
done! and also added `BitVec.reverse_reverse_eq` which seemed to be missing.